Core/src/main/java/com/plotsquared/core/queue/QueueCoordinator.java

```diff
@@ -71,33 +71,33 @@ public QueueCoordinator(@Nullable World world) {
     }
 
     /**
-     * Get a {@link ScopedQueueCoordinator} limited to the chunk at the specific chunk Coordinates
+     * Get a {@link ZeroedDelegateScopedQueueCoordinator} limited to the chunk at the specific chunk Coordinates
      *
      * @param x chunk x coordinate
      * @param z chunk z coordinate
-     * @return a new {@link ScopedQueueCoordinator}
-     * @deprecated Use {@link ScopedQueueCoordinator#getForChunk(int, int, int, int)}
+     * @return a new {@link ZeroedDelegateScopedQueueCoordinator}
+     * @deprecated Use {@link ZeroedDelegateScopedQueueCoordinator#getForChunk(int, int, int, int)}
      */
     @Deprecated(forRemoval = true, since = "6.6.0")
-    public ScopedQueueCoordinator getForChunk(int x, int z) {
+    public ZeroedDelegateScopedQueueCoordinator getForChunk(int x, int z) {
         if (getWorld() == null) {
             return getForChunk(x, z, PlotSquared.platform().versionMinHeight(), PlotSquared.platform().versionMaxHeight());
         }
         return getForChunk(x, z, getWorld().getMinY(), getWorld().getMaxY());
     }
 
     /**
-     * Get a {@link ScopedQueueCoordinator} limited to the chunk at the specific chunk Coordinates
+     * Get a {@link ZeroedDelegateScopedQueueCoordinator} limited to the chunk at the specific chunk Coordinates
      *
      * @param x chunk x coordinate
      * @param z chunk z coordinate
-     * @return a new {@link ScopedQueueCoordinator}
+     * @return a new {@link ZeroedDelegateScopedQueueCoordinator}
      * @since 6.6.0
      */
-    public ScopedQueueCoordinator getForChunk(int x, int z, int minY, int maxY) {
+    public ZeroedDelegateScopedQueueCoordinator getForChunk(int x, int z, int minY, int maxY) {
```

Comment:
That'll need to be deprecated in v6 while maintaining a since tag pointing to v7 on this branch.